### PR TITLE
Minimap props refactor: `miniMapBtn` & `zoomToolBar`

### PR DESCRIPTION
**Pull request by jitu5**
## Description

Resolves https://github.com/kedro-org/kedro-viz/pull/1969#issuecomment-2223916584 point 2 

## Development notes

In redux state `visible.miniMapBtn` not just control the visibility of mini map button but also all the zoom toolbar buttons 
Screenshot 2024-07-15 at 10 28 45 a m
 
 After PR changes `visible.miniMapBtn` only control the visibility of mini map button and `display.zoomToolbar` now controls the visibility of all the zoom toolbar buttons (zoom out, zoom in and zoom reset)

## Checklist

- [X] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Review by ravi-kumar-pilla — APPROVED** [review]
LGTM. Thank you

**Review by rashidakanchwala — APPROVED** [review]

File: src/components/app/app.js [hunk 1]
```diff
@@ -99,6 +99,8 @@ App.propTypes = {
     exportBtn: PropTypes.bool,
     pipelineBtn: PropTypes.bool,
     sidebar: PropTypes.bool,
+    zoomToolBar: PropTypes.bool,
+    miniMapBtn: PropTypes.bool,
```
Comment (ravi-kumar-pilla) [inline src/components/app/app.js:103]: @jitu5 I remember we thought of having miniMap only in display which will control both the button and the actual rectangular map area. Was there any change later ?

**Review by Huongg — APPROVED** [review]
👍 no questions from my side. Thank you @jitu5

**Comment by jitu5** [comment]
Closing this PR as it already merged with https://github.com/kedro-org/kedro-viz/pull/1992 to `main`.

**Pull request closed without merging**